src/ApplicationLogs/LogReader.cs

```diff
@@ -99,7 +99,7 @@ public static JObject TxLogToJson(Blockchain.ApplicationExecuted appExec)
                 }
                 catch (Exception ex)
                 {
-                    stack.Add("error: " + ex);
+                    stack.Add("error: " + ex.Message);
```

Comment:
```suggestion
                    stack.Add(item.GetInterface<object>().GetType().Name);
```

Reply:
This can break applicationlog getting the stack:
![3e8148970abed45ba6ef0e98c4161c3](https://github.com/neo-project/neo-modules/assets/38493437/d7793017-5599-4964-b6b8-4e3c653a8417)
But you can see it's `on-chain`:
![7723fcd8ecae99fdf1a629501952ada](https://github.com/neo-project/neo-modules/assets/38493437/50662d2b-2ae8-4015-ba05-05943ee41fff)